components/datatypes/device.schema.json

```diff
@@ -60,7 +72,11 @@
           "title": "Model number",
           "type": "string",
           "description": "The unique model number designation assigned by the manufacturer for this device. Model numbers are not versions, but unique identifiers that identify a particular model configuration. While the model for a particular phone might be 'iPhone 6S' the model number would be 'A1633', or 'A1634' based on configuration at the time of sale."
-        },
+        }
+      }
+    },
+    "screendimensions": {
```

Comment:
If we are splitting properties into their own definitions then I would suggest, we should still keep related properties together. Example, screenorientation and other screen properties could still exist in the same definition.

Reply:
@mjasrotia the downside of that is that the new field group will need to pull in all the fields even though it only requires `screendimensions`. Should we keep them as per @kevinlind 's definition in this case?

Reply:
@emdobrin , what you are saying is the reason we break properties into definitions and I would agree. I still think that just the screen properties won't explode in future, may be few more but should be good. We still could manage those right ? You can leave it the way you have it but even though we create definitions for properties, we should still try to keep the like/related properties under the same definition.

Reply:
@emdobrin , Are we good with the suggestion above ? This should not matter much as the field based workflow is coming in AEP-XDM where you would be able to choose individual properties from a field-group while creating schemas through UI. Please let us know if you are making any changes so that we can finally review and close this PR

Reply:
@mjasrotia , we would prefer to keep the screen dimensions and orientation properties separate if that's okay. I'm not working on any changes there.